plugin.go

```diff
@@ -261,3 +262,17 @@ func logParams(params map[string]string, paramsEnv []string) {
 		}
 	}
 }
+
+func getServerWithDefaults(server string) string {
+	if len(server) != 0 {
+		return server
+	}
+
+	host := os.Getenv("DRONE_SYSTEM_HOST")
```

Comment:
You should bind these variables to flags, we aren't using the env variables anywhere directly like that.